### PR TITLE
Fix double decrement while master standing down

### PR DESCRIPTION
**Pull request by tylerkaraszewski**
@coleaeason 

Fixes: https://github.com/Expensify/Expensify/issues/78456

This was tested by adding logging, and verifying in the cluster tests every time we decremented the _commandsInProgress below zero. This could happen before this change, but was not repeatable in 10 runs of the test after the change.

### CODE REVIEW

File: BedrockServer.cpp [hunk 1]
```diff
@@ -42,6 +42,15 @@ void BedrockServer::acceptCommand(SQLiteCommand&& command, bool isNew) {
         }
         SINFO("Queued new '" << command.request.methodLine << "' command from bedrock node, with " << _commandQueue.size()
               << " commands already queued.");
+
+        auto it = command.request.nameValueMap.find("Connection");
+        if (it != command.request.nameValueMap.end() && SIEquals(it->second, "forget")) {
+            // Forgotten commands are always "new". This is because when we escalate a one of these commands, we assume
```
Comment (coleaeason) [inline BedrockServer.cpp]: `a one of these`

**Comment by tylerkaraszewski** [comment]
@coleaeason updated

**Comment by tylerkaraszewski** [comment]
@coleaeason
